src/App/Patterns/content/Forms/index.js

```diff
@@ -9,32 +9,45 @@ import RadioButton from "@components/FormComponents/Radio";
 
 import LoginForm from "../../components/LoginForm/index";
 
-import { ComponentPreview } from "~/src/App/docutils";
+import { EditableComponentPreview } from "~/src/App/docutils";
 import { guidelinesElementsList, formsShowCase } from "./constants";
 import { hintTextExpander } from "@src/scripts/main";
 
 const radioOptions = [
     {
         label: "Option 1",
         id: "guidelines-options-1",
-        checked: true
+        checked: true,
     },
     {
         label: "Option 2",
-        id: "guidelines-options-2"
+        id: "guidelines-options-2",
     },
     {
         label: "Option 3",
-        id: "guidelines-options-3"
-    }
+        id: "guidelines-options-3",
+    },
```

Comment:
Why the dangling commas? 😨

Reply:
And why are they not present in my local? haha

Reply:
holy moly, I have the other branch in my local 😂 but still, why dangling commas?

Reply:
I answered above,
I enjoy the dangling-comma, I find it easier to move the code around with it, and to avoid having an error thrown each time I move an object or a property up or down, or create a new property,
but it is fine to remove it if you don't like them 😁

Reply:
Alright! I got a reprimand from Espen (Dj BeamBoom) telling me how conservative I am. He was sure I was going to have a downfall when turning 40 -> Start doing drugs and sleep with a lot of women 😂  Maybe being able to change myself, saying ok to dangling commas can help me avoid my imminent destruction 🤔